Swift/Replicator.swift

```diff
@@ -175,6 +175,35 @@ public final class Replicator {
         _impl.removeChangeListener(with: token._impl)
     }
     
+    /// Gets a set of document Ids, who have revisions pending push. This API is a snapshot and results
+    /// may change between the time the call was made and the time the call returns.
+    ///
+    /// - Returns: A  set of document Ids, each of which has one or more pending revisions
+    public func pendingDocumentIds() throws -> Set<String> {
+        var error: NSError?
+        let result = _impl.pendingDocumentIds(&error)
+        if let err = error {
+            throw err
+        }
+        
+        return result
+    }
+    
+    /// Checks if the document with the given ID has revisions pending push.  This API is a snapshot and
+    /// results may change between the time the call was made and the time the call returns.
+    ///
+    /// - Parameter documentID: The ID of the document to check
+    /// - Returns: true if the document has one or more revisions pending, false otherwise
+    public func isDocumentPending(_ documentID: String) throws -> Bool {
+        var error: NSError?
+        let result = _impl.isDocumentPending(documentID, error: &error)
```

Comment:
I have never used `NS_SWIFT_NOTHROW` before. Can you explain why we don't let the objective-c thrown the exception so we don't need to check the error?

Reply:
Never mind, I got your point here.